Sources/PT.PM.Common/TextUtils.cs

```diff
@@ -63,6 +65,64 @@ public static TextSpan Union(this IEnumerable<TextSpan> textSpans)
             return resultTextSpan;
         }
 
+        public static List<TextSpan> GetCombinedTextSpan(Ust ust, TextSpan[] locations)
```

Comment:
maybe it should be named `GetPreprocessedTextSpan`

Reply:
`GetAlignedTextSpans`. Definitely, it should be placed to another util class, maybe `UstUtils`.
`locations` -> `matchesLocations`.